### PR TITLE
Sanitize input

### PR DESCRIPTION
**Pull request by sebastian-berlin-wmse**
Initially strip leading and trailing whitespaces. Also, use
`newline=""` for open() calls, as by recommendation in the
documentation.

Bug: T241245

### CODE REVIEW

File: project_start.py [hunk 1]
```diff
@@ -99,6 +100,50 @@ def read_goals(tsv):
...
+        Dictionary or list to sanitize
+
+    Returns
+    -------
+    iterator
+        Copy of input with sanitized strings
+    """
+    sanitized = None
+    if isinstance(unsanitized, dict):
+        sanitized = {}
```
Comment (lokal-profil) [inline project_start.py]: Not a fan of dictionary comprehension?
```suggestion
        sanitized = {sanitize_string(k):sanitize_string(v) for (k,v) in unsanitized.items()}
```

File: project_start.py [hunk 2]
```diff
@@ -99,6 +100,50 @@ def read_goals(tsv):
...
+        for k, v in unsanitized.items():
+            sanitized[sanitize_string(k)] = sanitize_string(v)
+    elif isinstance(unsanitized, list):
+        sanitized = []
```
Comment (lokal-profil) [inline project_start.py]: Not a fan of list comprehension?
```suggestion
        sanitized = [sanitize_string(i) for i in unsanitized]
```

**Review by lokal-profil — APPROVED** [review]
Two questions but neither is blocking

**Comment by sebastian-berlin-wmse** [comment]
I like comprehensions so long as they are readable. I guess I haven't written a lot of Python lately :)

**Review by lokal-profil — APPROVED** [review]

**Pull request merged**